ui/src/app/cohort-search/overview/overview.component.css

```diff
@@ -6,14 +6,7 @@
   margin: 0.125rem 0;
 }
 
-#total-count {
-  text-align: right;
-  margin: 0;
-  flex: 1;
-  white-space: nowrap;
-}
-
-.save-cohort {
+h3 {
```

Comment:
Why did we drop the id selector here?  There's another `h3` in the component, although I don't think this affects it adversely at the moment.